packages/grafana-data/src/datetime/formats.test.ts

```diff
@@ -14,6 +14,25 @@ describe('Date Formats', () => {
       ''
     );
 
-    expect(format).toBe('MM/DD/YYYY, HH:mm:ss A');
+    expect(format).toBe('MM/DD/YYYY, hh:mm:ss A');
+  });
+});
+
+describe('Date Formats with hour12', () => {
```

Comment:
```suggestion
describe('Date Formats without hour12', () => {
```

Reply:
Probably not the best wording from me, ment 'with hour12 parameter' :)